src/app/components/TodoItem/todoItem.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, EventEmitter, Input, Output} from "@angular/core";
+
+
+@Component({
+  selector: 'app-todoItem',
```

Comment:
в ангулярі компоненти називаються через дефіс 
```suggestion
  selector: 'app-todo-item',
```

Reply:
+